src/common/options/crimson.yaml.in

```diff
@@ -78,6 +78,18 @@ options:
   level: dev
   desc: default logical address space reservation for seastore objects' metadata
   default: 16777216
+- name: seastore_full_integrity_check
+  type: bool
+  level: dev
+  desc: Whether seastore need to fully check the integrity of each extent,
+        non-full integrity check means the integrity check might be skipped
+        during extent remapping for better performance, disable with caution
+  default: false
+- name: seastore_max_data_allocation_size
```

Comment:
As full integrity check is disabled by default, I don't see a reason to set the limit, so should it be disabled?

Reply:
I think we need it anyway for now. Otherwise, the read amplification can be very large in case of reading a part of a relatively large extent. Am I right?

Reply:
See https://github.com/ceph/ceph/pull/50421, probably the sub-extent read cache can be prioritized to address the issue.

If it makes sense, let's add a TODO:
// TODO: disable once the sub-extent read cache is implemented.

Reply:
> See https://github.com/ceph/ceph/pull/50421, probably the sub-extent read cache can be prioritized to address the issue.
> 
> If it makes sense, let's add a TODO:
> // TODO: disable once the sub-extent read cache is implemented.
> 
> 

yeah, will do.